openpgp/packet/opaque.go

```diff
@@ -84,8 +84,8 @@ func (or *OpaqueReader) Next() (op *OpaquePacket, err error) {
 // OpaqueSubpacket represents an unparsed OpenPGP subpacket,
 // as found in signature and user attribute packets.
 type OpaqueSubpacket struct {
-	SubType  uint8
-	Contents []byte
+	SubType           uint8
+	LengthAndContents [][]byte //0: length, 1: contents
```

Comment:
```suggestion
	Contents []byte
	LengthAndContents []byte
```

Reply:
i thank maybe use bellow veriable
```
	SubType  uint8
	Contents []byte
         Length []byte
```
is more simple,